love/src/components/EnvironmentSummary/Cartoons/WindDirection.jsx

```diff
@@ -0,0 +1,61 @@
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import ArrowIcon from 'components/icons/ArrowIcon/ArrowIcon';
+import styles from './WindDirection.module.css';
+
+export class WindDirection extends Component {
+  static propTypes = {};
+
+  render() {
+    // TODO: calcular con escala
+    const minSpeed = 0;
+    const maxSpeed = 100;
+    const currentSpeed = 50;
+    const maxArrowHeight = 500;
+    const arrowHeight = (currentSpeed / maxSpeed) * maxArrowHeight;
```

Comment:
I think this is missing to connect the respective telemetry.

Reply:
You're right! Thanks!! Resolved!